test/vm_tests/transformer/impl_generation_test.dart

```diff
@@ -652,6 +731,33 @@ main() {
           });
         });
       });
+
+      group('static meta field', () {
```

Comment:
Can we get tests that verify the validation in the declaration parsing for Props/State meta is working properly?